x-pack/plugins/apm/public/components/app/Main/routeConfig.tsx

```diff
@@ -64,49 +90,6 @@ export const routes: Route[] = [
       defaultMessage: 'Errors'
     })
   },
-  {
```

Comment:
If we wrap all of our main routes in one `<Switch>`, we don't need to worry about the "switchRoutes" logic and can simplify the routes config, and no longer need to "flatten" the routes before giving the routes config to the breadcrumbs component, etc.